### DISCUSSION TITLE
RemapPal won't work in FightFX Sprites

### DISCUSSION THREAD

**Discussion by Ksylatarsh** [ikemen-engine/Ikemen-GO#1021] [Feature Request] [answered]
### Describe the bug

RemapPal just doesn't work on FightFX sprites.

### To Reproduce

Change the anim from any character/helper to one from the FightFX and add a RemapPal to it.

[State RemapPal]
Type = RemapPal
Trigger1 = 1
Source = -1,0
Dest = 1,1

### Expected behavior

Well, it should put a palette from the FightFX.sff to the current sprites.

### Engine Version (or source code date)

0.98.2

### Operating system

Windows

**Comment by beterhans** [comment, marked as answer]
i guess same as mine
https://github.com/ikemen-engine/Ikemen-GO/issues/1001

**Comment by K4thos** [comment]
yeah, a duplicate, so let's close this one

**Comment by K4thos** [comment]
not a bug or regression since it never used to work and it's not advertised as a feature in the documentation. Also doesn't work in mugen. I'm converting this ticket to discussion page.
